src/Microsoft.Health.Fhir.CosmosDb.Initialization/Features/Storage/StoredProcedures/DataPlaneStoredProcedureInstaller.cs

```diff
@@ -9,24 +9,26 @@
 using System.Threading.Tasks;
 using EnsureThat;
 using Microsoft.Azure.Cosmos;
-using Microsoft.Health.Fhir.CosmosDb.Features.Storage.Versioning;
+using Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage.StoredProcedures;
 
-namespace Microsoft.Health.Fhir.CosmosDb.Features.Storage.StoredProcedures
+namespace Microsoft.Health.Fhir.CosmosDb.Initialization.Features.Storage.StoredProcedures
 {
-    public class StoredProcedureInstaller : ICollectionUpdater
+    public sealed class DataPlaneStoredProcedureInstaller : IStoredProcedureInstaller
     {
-        private readonly IEnumerable<IStoredProcedure> _storedProcedures;
+        private readonly IEnumerable<IStoredProcedureMetadata> _storeProceduresMetadata;
 
-        public StoredProcedureInstaller(IEnumerable<IStoredProcedure> storedProcedures)
+        // TODO: refactor constructor to have dependency on container
```

Comment:
I believe that now we are implementing the ARM Collection Setup, we'll face some issues as IStoredProcedureInstaller has a dependency to Container, and Container is not available through ARM. 
Let's sync about this.

Reply:
As discussed for now we are good with this approach